src/Uno.UI/Microsoft/UI/Xaml/Controls/CalendarView/CalendarDatePicker.Properties.cs

```diff
@@ -0,0 +1,248 @@
+using System;
+
+using DateTime = System.DateTimeOffset;
+using Calendar = Windows.Globalization.Calendar;
+using DayOfWeek = Windows.Globalization.DayOfWeek;
+
+namespace Windows.UI.Xaml.Controls
+{
+	partial class CalendarDatePicker
+	{
+		public static readonly DependencyProperty CalendarIdentifierProperty = DependencyProperty.Register(
+			"CalendarIdentifier", typeof(string), typeof(CalendarDatePicker), new PropertyMetadata(default(string)));
```

Comment:
Should default to GregorianCalendar, like the CalendarView propertiy

Reply:
In the WinUI code it's set in the `SetPropertyDefaultValue()`

Reply:
Looking at `SetPropertyDefaultValue()`, I see it create a GergorianCalendar but it doesn't seem to be setting the CalendarIdentifierProperty anywhere